### ISSUE TITLE
How to use this

### ISSUE THREAD

**Issue by ibeckermayer** [Miej/GoDeeper#4]
I'm trying to use this image, but I must be doing something wrong. I don't quite understand your instructions in the quick start guide. I have an AWS account, and I've been able to find this image in the community images and launch it. But after that, I don't understand what we're meant to do with:

User credentials: username: icarus password: changetheworld

What I've done, so far, is log into the instance using SSH from my mac. However, once I do that, it seems as if none of the packages you've installed are there. For example, conda, caffe, tensorflow, etc. cannot be imported in python. Is there some problem with using SSH? I'm confused as to why this isn't working.

**Comment by Miej** [comment]
Do `$ su icarus`.  The packages are installed user-specific.  I plan to update this for the next release.  Let me know if that works for you.

**Comment by ibeckermayer** [comment]
Yes! Got it to work. I was also able to install nolearn and lasagne easily, and they are running smoothly, which is something you were looking to add in the next release. The only issue that I've found is that installing pylearn2 is buggy, which lasagne needs for its cuda_covnet layers. Fortunately I was able to get around this by simply using the cuDNN layers instead, which are also GPU optimized.
